snap/hooks/configure

```diff
@@ -5,10 +5,10 @@
 # `sudo snap get dcgm`.
 if [ -z "$(snapctl get nv-hostengine-port)" ]; then
     # Setting to empty string for the nv-hostengine binary to use the default port. (5555)
```

Comment:
the comments should now be removed if we're explicitly setting the default option values